lightningd/jsonrpc.c

```diff
@@ -709,14 +709,10 @@ static void replace_command(struct rpc_command_hook_payload *p,
 		goto fail;
 	}
 
-	// deprecated phase to give the possibility to all to migrate and stay safe
-	// from this more restrictive change.
-	if (!deprecated_apis) {
-		const jsmntok_t *jsonrpc = json_get_member(buffer, replacetok, "jsonrpc");
-		if (!jsonrpc || jsonrpc->type != JSMN_STRING || !json_tok_streq(buffer, jsonrpc, "2.0")) {
-			bad = "jsonrpc: \"2.0\" must be specified in the request";
-			goto fail;
-		}
+	jsonrpc = json_get_member(buffer, replacetok, "jsonrpc");
+	if (!jsonrpc || jsonrpc->type != JSMN_STRING || !json_tok_streq(buffer, jsonrpc, "2.0")) {
+		bad = "jsonrpc: \"2.0\" must be specified in the request";
```

Comment:
I think this broke the BTCPay server integration… cc @NicolasDorier

Reply:
The broken part is the RPC library used inside it